1-WebApp-OIDC/1-1-MyOrg/WebApp-OpenIDConnect-DotNet.csproj

```diff
@@ -25,12 +25,9 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
     <PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="3.0.0" />
+    <PackageReference Include="Microsoft.Identity.Web" Version="0.1.0-localbuild" />
```

Comment:
Hmm, should version be `0.1.0-preview` or something?

Reply:
I know. I wanted to test everything w/the nuget package w/the current release build. we will have to run a new package w/the correct name.